docs/zope4/migration/zmi.rst

```diff
@@ -24,6 +24,13 @@ Example to use the info icon (i in a circle)::
 
     zmi_icon = 'fas fa-info-circle'
 
+If your product does not need to show a form when adding an object in the ZMI
+using the `Select type to add` dropdown, which only applies to a small
```

Comment:
I think the part between the commas ("which only applies to a small …") make the sentence harder to read. I am not sure if it provides additional value to the sentence. Maybe it could be removed.

Reply:
I have changed the wording to be clearer now. I want to make sure people realize this only affects very few products.